android/app/src/main/java/updated/mysterium/vpn/common/countries/CountriesUtil.kt

```diff
@@ -1,21 +1,262 @@
 package updated.mysterium.vpn.common.countries
 
-import updated.mysterium.vpn.model.settings.ResidentCountry
-import java.util.Locale
+import updated.mysterium.vpn.model.settings.CountryISO
+import java.util.*
 
-class CountriesUtil {
+object CountriesUtil {
 
-    fun getAllCountries(): List<ResidentCountry> {
-        val countries = mutableListOf<ResidentCountry>()
+    fun getAllResidentCountries(): List<CountryISO> {
+        val countries = mutableListOf<CountryISO>()
         val isoCountries = Locale.getISOCountries()
         for (country in isoCountries) {
             val locale = Locale("en", country)
             val code = locale.country
-            val name = locale.getDisplayCountry(Locale.ENGLISH) // Replace for current selected language
+            val name =
+                locale.getDisplayCountry(Locale.ENGLISH) // Replace for current selected language
             if (code.isNotEmpty() && name.isNotEmpty()) {
-                countries.add(ResidentCountry(name, code))
+                countries.add(CountryISO(name, code))
             }
         }
         return countries.sortedBy { it.fullName }
     }
+
+    fun getAllPaymentCountries() = listOf(
```

Comment:
Instead of hardcoding country name you get get name by country code:
```
Locale("","NL").getDisplayCountry()
```

Reply:
Done